ferrocene/doc/target-names.toml

```diff
@@ -7,18 +7,21 @@
 # In the documents, as long as the ferrocene_qualification extension is added,
 # you can refer to them with :target:`triple`.
 
-aarch64-unknown-none      = "Armv8-A bare-metal"
-thumbv7em-none-eabi       = "Armv7E-M bare-metal (soft-float)"
-thumbv7em-none-eabihf     = "Armv7E-M bare-metal (hard-float)"
-armv8r-none-eabihf        = "Armv8-R bare-metal (hard-float)"
-armv7r-none-eabihf        = "Armv7-R bare-metal (hard-float)"
-armebv7r-none-eabihf      = "Armv7-R bare-metal (hard-float, big-endian)"
-wasm32-unknown-unknown    = "WASM bare-metal"
-x86_64-unknown-linux-gnu  = "x86-64 Linux (glibc)"
-aarch64-unknown-linux-gnu = "Armv8-A Linux (glibc)"
-aarch64-apple-darwin      = "Apple Silicon macOS"
-x86_64-apple-darwin       = "x86-64 macOS"
-x86_64-pc-windows-msvc    = "x86-64 Windows"
-aarch64-unknown-nto-qnx710 = "Armv8-A QNX 7.1.0"
-x86_64-pc-nto-qnx710       = "x86-64 QNX 7.1.0"
+aarch64-unknown-none        = "Armv8-A bare-metal"
+thumbv7em-none-eabi         = "Armv7E-M bare-metal (soft-float)"
+thumbv7em-none-eabihf       = "Armv7E-M bare-metal (hard-float)"
+armv8r-none-eabihf          = "Armv8-R bare-metal (hard-float)"
+"thumbv8m.base-none-eabi"   = "Armv8-M Baseline bare-metal"
+"thumbv8m.main-none-eabi"   = "Armv8-M Mainline with Soft Float ABI"
```

Comment:
Sorry, I've added an inconsistency here by accident.

The other targets say "(soft-float)" etc and these ones should match instead of saying "with Soft Float ABI". Same for hard float.

Reply:
Fixed!